### PR TITLE
RS: 7.4.6 July release docs

### PR DESCRIPTION
**Pull request by rrelledge**
DOC-3484

To be added:
- SHA-256 checksums
- Build number

### CODE REVIEW

File: content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+
+- [RediSearch 2.8.13]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.8-release-notes.md" >}})
+
+- [RedisJSON 2.6.10]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisjson/redisjson-2.6-release-notes.md" >}})
+
+- [RedisTimeSeries 1.10.12]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redistimeseries/redistimeseries-1.10-release-notes.md" >}})
+
+- [RedisBloom 2.6.12]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisbloom/redisbloom-2.6-release-notes.md" >}})
+
+- [RedisGears 2.0.20 preview](https://github.com/RedisGears/RedisGears/releases/tag/v2.0.20-m21)
```
Comment (LiorKogan) [inline content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md:52]: Please add a note - RedisGears preview will not be promoted to GA and will be removed from the next release.

File: content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+
+- [RediSearch 2.6.18]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.6-release-notes.md" >}})
+
+- [RedisJSON 2.4.9]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisjson/redisjson-2.4-release-notes.md" >}})
+
+- [RedisTimeSeries 1.8.13]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redistimeseries/redistimeseries-1.8-release-notes.md" >}})
+
+- [RedisBloom 2.4.9]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisbloom/redisbloom-2.4-release-notes.md" >}})
+
+- [RedisGraph v2.10.15]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisgraph/redisgraph-2.10-release-notes.md" >}})
```
Comment (LiorKogan) [inline content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md:64]: Please add a note - RedisGraph end-of-life has been announced and will be removed from future releases.

File: content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+
+- (CVE-2023-41056) In some cases, Redis may incorrectly handle resizing of memory buffers, which can result in incorrect accounting of buffer sizes and lead to heap overflow and potential remote code execution.
+
+- (CVE-2023-41053) Redis does not correctly identify keys accessed by `SORT_RO` and, as a result, may grant users executing this command access to keys that are not explicitly authorized by the ACL configuration. (Redis 7.0.13)
+
+- (CVE-2023-36824) Extracting key names from a command and a list of arguments may, in some cases, trigger a heap overflow and result in reading random heap memory, heap corruption, and potentially remote code execution. Specifically: using `COMMAND GETKEYS*` and validation of key names in ACL rules. (Redis 7.0.12)
+
+- (CVE-2023-28856) Authenticated users can use the `HINCRBYFLOAT` command to create an invalid hash field that will crash Redis on access. (Redis 7.0.11)
+
+- (CVE-2023-28425) Specially crafted `MSETNX` command can lead to assertion and denial-of-service. (Redis 7.0.10)
```
Comment (andy-stark-redis) [inline content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md:246]:
```suggestion
- (CVE-2023-28425) Specially crafted `MSETNX` commands can lead to assertion and denial-of-service. (Redis 7.0.10)
```

File: content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+
+- (CVE-2023-25155) Specially crafted `SRANDMEMBER`, `ZRANDMEMBER`, and `HRANDFIELD` commands can trigger an integer overflow, resulting in a runtime assertion and termination of the Redis server process. (Redis 7.0.9)
+
+- (CVE-2023-22458) Integer overflow in the Redis `HRANDFIELD` and `ZRANDMEMBER` commands can lead to denial-of-service. (Redis 7.0.8)
+
+- (CVE-2022-36021) String matching commands (like `SCAN` or `KEYS`) with a specially crafted pattern to trigger a denial-of-service attack on Redis, causing it to hang and consume 100% CPU time. (Redis 7.0.9)
```
Comment (andy-stark-redis) [inline content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md:252]:
```suggestion
- (CVE-2022-36021) String matching commands (like `SCAN` or `KEYS`) with a specially crafted pattern to trigger a denial-of-service attack on Redis can cause it to hang and consume 100% CPU time. (Redis 7.0.9)
```

File: content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+
+Redis 6.2.x:
+
+- (CVE-2023-28856) Authenticated users can use the `HINCRBYFLOAT` command to create an invalid hash field that will crash Redis on access. (Redis 6.2.12)
+
+- (CVE-2023-25155) Specially crafted `SRANDMEMBER`, `ZRANDMEMBER`, and `HRANDFIELD` commands can trigger an integer overflow, resulting in a runtime assertion and termination of the Redis server process. (Redis 6.2.11)
+
+- (CVE-2023-22458) Integer overflow in the Redis `HRANDFIELD` and `ZRANDMEMBER` commands can lead to denial-of-service. (Redis 6.2.9)
+
+- (CVE-2022-36021) String matching commands (like `SCAN` or `KEYS`) with a specially crafted pattern to trigger a denial-of-service attack on Redis, causing it to hang and consume 100% CPU time. (Redis 6.2.11)
```
Comment (andy-stark-redis) [inline content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md:274]:
```suggestion
- (CVE-2022-36021) String matching commands (like `SCAN` or `KEYS`) with a specially crafted pattern to trigger a denial-of-service attack on Redis can cause it to hang and consume 100% CPU time. (Redis 6.2.11)
```

File: content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md [hunk 6]
```diff
@@ -0,0 +1,318 @@
...
+
+Redis 6.0.x:
+
+- (CVE-2022-24834) A specially crafted Lua script executing in Redis can trigger a heap overflow in the cjson and cmsgpack libraries, and result in heap corruption and potentially remote code execution. The problem exists in all versions of Redis with Lua scripting support, starting from 2.6, and affects only authenticated and authorized users. (Redis 6.0.20)
+
+- (CVE-2023-28856) Authenticated users can use the `HINCRBYFLOAT` command to create an invalid hash field that will crash Redis on access. (Redis 6.0.19)
+
+- (CVE-2023-25155) Specially crafted `SRANDMEMBER`, `ZRANDMEMBER`, and `HRANDFIELD` commands can trigger an integer overflow, resulting in a runtime assertion and termination of the Redis server process. (Redis 6.0.18)
+
+- (CVE-2022-36021) String matching commands (like `SCAN` or `KEYS`) with a specially crafted pattern to trigger a denial-of-service attack on Redis, causing it to hang and consume 100% CPU time. (Redis 6.0.18)
```
Comment (andy-stark-redis) [inline content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md:316]:
```suggestion
- (CVE-2022-36021) String matching commands (like `SCAN` or `KEYS`) with a specially crafted pattern to trigger a denial-of-service attack on Redis can cause it to hang and consume 100% CPU time. (Redis 6.0.18)
```

**Review by andy-stark-redis — APPROVED** [review]
Few minor suggestions but otherwise the language LGTM.

File: content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md (hunk 1 above)
Comment (rrelledge) [reply content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md]: Added

File: content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md (hunk 2 above)
Comment (rrelledge) [reply content/operate/rs/release-notes/rs-7-4-6-releases/rs-7-4-6-tba.md]: Added

**Review by yoavredis — APPROVED** [review]
LGTM, 10x

**Pull request merged**